doczrc.js

```diff
@@ -5,6 +5,7 @@ export default {
     menu: [
         'Entur SDK',
         'Getting Started',
+        'Advanced',
```

Comment:
Ha gjerne advanced heilt til slutt, sidan det er for spesiell bruk.

Reply:
Fikset!